SpatialGDK/Source/SpatialGDK/Public/Schema/RPCPayload.h

```diff
@@ -76,6 +77,7 @@ struct RPCPayload
 	TOptional<uint64> Id;
 	TArray<uint8> PayloadData;
 	TraceKey Trace = InvalidTraceKey;
+	FSpatialGDKSpanId SpanId; // Not serialized
```

Comment:
I think this is good. As a passing thought. I wonder if it would be "nicer" to have a new class containing an RPCPayload and a FSpatialGDKSpanId. It would be a lot of annoying code changes. I just wonder if there is any utility in it.

Reply:
Will take a look!